build.sh

```diff
@@ -189,53 +189,60 @@ restore_optdata()
 
 generate_event_logging_sources()
 {
-    if [ $__SkipCoreCLR == 1 ]; then
-        return
-    fi
+    __OutputDir=$1
+    __ConsumingBuildSystem=$2
 
-# Event Logging Infrastructure
-    __GeneratedIntermediate="$__IntermediatesDir/eventing"
-    __GeneratedIntermediateEventProvider="$__GeneratedIntermediate/eventprovider"
-    __GeneratedIntermediateEventPipe="$__GeneratedIntermediate/eventpipe"
+    __OutputIncDir="$__OutputDir/src/inc"
+    __OutputEventingDir="$__OutputDir/eventing"
+    __OutputEventProviderDir="$__OutputEventingDir/eventprovider"
+
+    echo "Laying out dynamically generated files consumed by $__ConsumingBuildSystem"
+    echo "Laying out dynamically generated Event test files, etmdummy stub functions, and external linkages"
 
     __PythonWarningFlags="-Wall"
     if [[ $__IgnoreWarnings == 0 ]]; then
         __PythonWarningFlags="$__PythonWarningFlags -Werror"
     fi
 
+    $PYTHON -B $__PythonWarningFlags "$__ProjectRoot/src/scripts/genEventing.py" --inc $__OutputIncDir --dummy $__OutputIncDir/etmdummy.h --man "$__ProjectRoot/src/vm/ClrEtwAll.man" --testdir "$__OutputEventProviderDir/tests"
+    if [[ $? != 0 ]]; then
+        exit
```

Comment:
Shouldn't all these `exit` be `exit 1` or `exit $?`?

Reply:
@BruceForstall For consistency with other parts of build.sh, I do `exit 1`